industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/itk/PREPARATION_DETAILS.md

```diff
@@ -0,0 +1,8 @@
+## System preparation details
+
+_For example:_
+
+* All prepration was carried out using maestro and text editors.
+* Converted N-terminal H atom to H3.
+* Converted C-terminal O atom to OXT.
+* The
```

Comment:
I think this bullet point is potentially missing the ligand prep details?

Reply:
done!